tests/framework/microvm.py

```diff
@@ -316,7 +316,11 @@ def _validate_api_response_times(self):
     def firecracker_version(self):
         """Return the version of the Firecracker executable."""
         _, stdout, _ = utils.run_cmd(f"{self._fc_binary_path} --version")
-        return re.match(r"^Firecracker v(.+)", stdout.partition("\n")[0]).group(1)
+        # The 1st line will be `Running Firecracker vX.Y.Z`
```

Comment:
Do we log by default to the console? The current version of Firecracker does not output anything:

```sh
./build/cargo_target/x86_64-unknown-linux-musl/release/firecracker --version
Firecracker v1.5.0-dev

Supported snapshot data format versions:
...
```

If we log to the console by default, can we disable that behavior for --version / --describe-snapshot / --help?

Reply:
So now when starting Firecracker it will always print (with the version changing):
```
Running FIrecracker v1.5.0-dev
```
This does make `--version` somewhat redundant.

I think for now to avoid a more breaking changing it is best to avoid changing `--version` but it may make sense to adjust this in the future.

I have a commit on top of my changes that removes the `Running Firecracker` log which I think is the best approach https://github.com/JonathanWoollett-Light/firecracker/commit/ee883c602c60b5b70ec6e46a95c6945e04ca006d